handbook/product/product_management/direction_template.md

```diff
@@ -0,0 +1,90 @@
+
+> As you write this document, keep in mind that focusing on problems to solve more than lists of features tends to be a better way to approach communicating with customers, as it leaves room for learning between what you know now and what you deliver in the end. This is not to say you never talk about features, but especially as you get [further out in planning](https://medium.com/@jobv/the-nearsighted-roadmap-57fa57b5906a) the blend of features vs. problems to solve should move more towards the latter.
+
+---
+
+# MyProductArea Direction
+
+This page outlines the vision, strategy and goals of the MyProductArea team. You can use this section to describe what your scope is, and where your product is today.
+
+Quicklinks:
+
+- [MyProductArea Backlog](https://about.sourcegraph.com/) <!-- Update links with your page -->
+- [Latest demo](https://about.sourcegraph.com/)
+- [Documentation](https://about.sourcegraph.com/)
+
+# Vision
+
+#### 1 Year vision
+
+> Add a sentence or two here indicating what the most important things you want to achieve/problems you want to solve in the next year are. These should in most cases move your three and ten year visions forward.
+
+#### 3 Year vision
+
+> Add a sentence or two here describing what your product will be like in three years, also focusing on the problems you'll solve and the value you'll deliver. This should be grounded in reality, but three years is a lot of time - be ambitious!
+
+#### 10 year vision
+
+> A sentence or two on your ten year vision is where you can really go wild and describe the problems you want to solve and solutions you want to deliver unbound by limitations of technology or scope. Be inspiring and look to the our [10 and 30 year strategies](../../company/strategy.md#sourcegraph-strategy) for ideas.
+
+## Mission
+
+What is the team's reason for existing? What is the fundamental value they aim to provide?
+
+# Where we are now
+
+- Where is the team's area of ownership in terms of its maturity? Is it new and basic, or complete and lovable? Are different features at different levels?
+- What did we achieve in the last few months?
+- What key learnings did we recently make?
+- What is on the critical path for growth?
+- How does the product fit within Sourcegraph as a whole?
+
+## Competitive landscape
+
+- Who is the competition for this product area?
+- What recent developments in competition is impacting sales, marketing and product strategy?
+
+## Analyst landscape
+
+- Are there analysts tracking this product area?
+- How are analysts positioning the product? What are areas of improvement?
+
+# 3 year strategy and themes
+
+## Strategy
+
+What is your strategy defined in a few steps?
+
+## Themes
+
+What are high level themes or principles we use to guide prioritisation? What are we focusing on in the next few years?
```

Comment:
I am trying to differentiate themes as part of the three year strategy from the vision section above. What do you think?

```suggestion
What are high level themes or principles you are using to guide prioritization within your three year strategy? What are the groupings of features/problems to solve that you are focusing on in that period?
```